docs/data/material/getting-started/templates/sign-in-side/SignInSide.tsx

```diff
@@ -39,90 +106,184 @@ export default function SignInSide() {
-            </Typography>
-            <Box component="form" noValidate onSubmit={handleSubmit} sx={{ mt: 1 }}>
-              <TextField
-                margin="normal"
-                required
-                fullWidth
-                id="email"
-                label="Email Address"
-                name="email"
-                autoComplete="email"
-                autoFocus
-              />
-              <TextField
-                margin="normal"
-                required
-                fullWidth
-                name="password"
-                label="Password"
-                type="password"
-                id="password"
-                autoComplete="current-password"
-              />
-              <FormControlLabel
-                control={<Checkbox value="remember" color="primary" />}
-                label="Remember me"
-              />
-              <Button
-                type="submit"
-                fullWidth
-                variant="contained"
-                sx={{ mt: 3, mb: 2 }}
+          />
+          <Grid item xs={12} sm={6} md={4}>
+            <Box
+              sx={{
+                mt: 4,
+                mx: 4,
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'center',
+                gap: 6,
+              }}
+            >
+              <Box
+                sx={{
+                  display: 'flex',
+                  justifyContent: 'space-between',
+                  width: '100%',
+                  ml: '-18px',
+                  alignItems: 'center',
+                }}
+              >
+                <img
+                  src={
+                    'https://assets-global.website-files.com/61ed56ae9da9fd7e0ef0a967/61f12e6faf73568658154dae_SitemarkDefault.svg'
+                  }
+                  style={logoStyle}
+                  alt="Sitemark's logo"
+                />
+                <ToggleColorMode mode={mode} toggleColorMode={toggleColorMode} />
+              </Box>
+              <Typography
+                component="h1"
+                variant="h4"
+                sx={{ fontSize: 'clamp(2rem, 10vw, 2.5rem)' }}
               >
                 Sign In
-              </Button>
-              <Grid container>
-                <Grid item xs>
-                  <Link href="#" variant="body2">
-                    Forgot password?
-                  </Link>
-                </Grid>
-                <Grid item>
-                  <Link href="#" variant="body2">
-                    {"Don't have an account? Sign Up"}
+              </Typography>
+              <Box
+                component="form"
+                noValidate
+                onSubmit={handleSubmit}
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  mt: 1,
+                  maxWidth: 400,
+                  alignItems: 'center',
+                  width: '100%',
```

Comment:
```suggestion
                  mt: 1,
                  maxWidth: 400,
                  width: '100%',
                  display: 'flex',
                  flexDirection: 'column',
                  alignItems: 'center',
```

Nit for a better code organization: flex-related properties together and container-related stuff together